backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/RemoveDiskCommand.java

```diff
@@ -728,4 +733,17 @@ private Guid getQuotaId() {
     @Override
     public void addQuotaPermissionSubject(List<PermissionSubject> quotaPermissionList) {
     }
+
+    private DiskStorageType getDiskStorageType() {
+        if (getParameters().getDiskStorageType() == null) {
+            getParameters().setDiskStorageType(getDisk().getDiskStorageType());
+            persistCommandIfNeeded();
```

Comment:
it's probably better to put it at the end of executeCommand, in case there will be additional changes to the parameters later